### PR TITLE
Apply scalafmt & add py entry point

### LINKED ISSUE #47: Create a single entry point for both connect and classic ways of running tsumugi

### PR DESCRIPTION
**Pull request by SemyonSinchenko**
- add py entry point
- simplify connect plugin to allows a simple support of multiversions
- apply spotless:apply
- run tests in CI

Close #47

### CODE REVIEW

**Comment by SemyonSinchenko** [comment]
@zeotuan Can you please take a look when you have time?

File: tsumugi-server/.scalafmt.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version = 3.8.2
+version = 3.7.3
```
Comment (zeotuan) [inline tsumugi-server/.scalafmt.conf:1]: I am curious why we need to downgrade scalafmt verison

**Review by zeotuan — APPROVED** [review]

File: tsumugi-server/.scalafmt.conf (hunk 1 above)
Comment (SemyonSinchenko) [reply tsumugi-server/.scalafmt.conf:1]: There was a very strange error with an advice to downgrade

Comment (zeotuan) [reply tsumugi-server/.scalafmt.conf:1]: You happen to keep the log of what's was wrong? I am not oppose using a older version but want to see what the error was

Comment (SemyonSinchenko) [reply tsumugi-server/.scalafmt.conf:1]: It seems to me that was something like maven wasnt able to resolve dependencies... Let me try again

Comment (SemyonSinchenko) [reply tsumugi-server/.scalafmt.conf]: Ok, I resolved it!
